windows/deploy_windows.ps1

```diff
@@ -256,6 +257,22 @@ function Build-App-Variants
 # Build Windows installer
 Function Build-Installer
 {
+    param(
+        [string] $BuildOption
+    )
+
+    # Building the installer with JACK support on Windows is using a separate nsi script which creates a separate installer.
+    # The installer with JACK on Windows is made available for more users who know JACK, ASIO will be kept as the mainstream version.
+    # When not building with JACK support, the installer is created with ASIO support.
+    if ($BuildOption -Eq "jackonwindows")
+    {
```

Comment:
I'd like the "installer suffix" passed in.  If it's not empty, set the name to "installer_${InstallerSuffix}.nsi" otherwise "installer.nsi".  That's because the portaudio installer will also be separate and should have a different name.

Reply:
No longer using a separate installer script.